### DISCUSSION TITLE
MIT資訊科學家Sinan Aral針對假新聞在Twitter上的傳播模式做的研究

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#691] [General]
因為上一則發了有關英國女子死於演算法的文章，
所以我想分享一下我的新發現，在這個TED演講中，MIT資訊科學家Sinan Aral針對2016-2017年間假新聞在Twitter上的傳播模式進行了研究，假新聞的傳播速度非常驚人，是真消息的6倍！而假新聞在傳播的廣度、深度也是比真新聞來得迅速！
而在不同的情緒上，「信任、喜樂、期待、難過、憤怒、害怕、噁心、驚喜」中，噁心與驚喜的假消息獲得的互動數量較多，從這個研究可發現，帶有負面情緒的新聞互動數較多。

由此可知有多數的人們所接觸到的悲觀的新聞資訊較多，若根據演算法，那麽這些接觸負面新聞的人們將會收到更多悲觀新聞或資訊的推送，是很可怕的一件事！

影片連結：https://www.youtube.com/watch?v=-7ORAKULel4&t=587s

**Comment by Vincent20030314** [comment]
演算法在幫助人的同時可能也會傷害到他人，希望未來可以改善。

**Comment by HsuChen88** [comment]
我在想會不會因為假消息通常都會有比較吸睛的標題，譬如較不可能在現實中發生的事。或是從另一個角度說，通常就是在大家關注的議題中，越容易有假消息的出現。

**Comment by yjpxi** [comment]
以ig為例，如果我們很常滑一些負面的視頻，像是分手文、自殺文類的，之後再看視頻的時候會很常看到類似的文章，這是因為演算法識別出我們對某類負面新聞有興趣，會推送更多類似內容，從而加劇回聲室效應，會讓我們沉浸在單一觀點的資訊泡沫中，增加社會極化。
